server/lib/dynamoSessionStore.js

```diff
@@ -0,0 +1,335 @@
+/*!
+ * Connect - DynamoDB
+ * Copyright(c) 2020 introvert.com LLC <support@introvert.com>
```

Comment:
We don't need their license :P this is our stuff now.

Reply:
YES!